src/main/java/org/gridsuite/modification/server/dto/ModificationInfos.java

```diff
@@ -84,6 +85,12 @@ public class ModificationInfos {
     @Schema(description = "Modification flag")
     private Boolean stashed;
 
+    @Schema(description = "Message type")
+    private String messageType;
+
+    @Schema(description = "Message values")
+    private String messageValues;
```

Comment:
why do you keep a raw json version of `getMapMessageValues()` instead of simply do once the conversion dto.valuesMap <--> entity.messageValues methods?